Add macOS and Debian install (#28)
lena-larionova/docs.konghq.com: app/gateway/2.6.x/install-and-run/debian.md

---

> id: page-install-method
> title: Install Kong Gateway on Debian
> header_title: Debian Installation
> header_icon: /assets/images/icons/icn-installation.svg
> breadcrumbs:
>   Installation: /install
> ---

Reviewer: ```suggestion
title: Install Kong Gateway on Debian
badge: oss
---
```
You can remove all of this metadata, it's legacy content now.

> 

Reviewer: <!-- Banner with links to latest downloads -->
<!-- The install-link and install-listing-link classes are used for tracking, do not remove -->
```suggestion
{:.install-banner}
> Download the latest {{site.ce_product_name}} {{page.kong_version}} package for Debian:
> * [8 Jessie]({{ site.links.download }}/gateway-2.x-debian-jessie/pool/all/k/kong/kong_{{page.kong_versions[page.version-index].ce-version}}_amd64.deb){:.install-link}
> * [9 Stretch]({{ site.links.download }}/gateway-2.x-debian-stretch/pool/all/k/kong/kong_{{page.kong_versions[page.version-index].ce-version}}_amd64.deb){:.install-link}
> * [10 Buster]({{ site.links.download }}/gateway-2.x-debian-buster/pool/all/k/kong/kong_{{page.kong_versions[page.version-index].ce-version}}_amd64.deb){:.install-link}
> * [11 Bullseye]({{ site.links.download }}/gateway-2.x-debian-bullseye/pool/all/k/kong/kong_{{page.kong_versions[page.version-index].ce-version}}_amd64.deb){:.install-link}
> (latest version: {{page.kong_versions[page.version-index].ce-version}})
>
> <br>
> <span class="install-subtitle">View the list of all 2.x packages for
> [8 Jessie]({{ site.links.download }}/gateway-2.x-debian-jessie/pool/all/k/kong/kong_{{page.kong_versions[page.version-index].ce-version}}_amd64.deb){:.install-listing-link}, 
> [9 Stretch]({{ site.links.download }}/gateway-2.x-debian-stretch/pool/all/k/kong/kong_{{page.kong_versions[page.version-index].ce-version}}_amd64.deb){:.install-listing-link}, 
> [10 Buster]({{ site.links.download }}/gateway-2.x-debian-buster/pool/all/k/kong/kong_{{page.kong_versions[page.version-index].ce-version}}_amd64.deb){:.install-listing-link}, or 
> [11 Bullseye]({{ site.links.download }}/gateway-2.x-debian-bullseye/pool/all/k/kong/kong_{{page.kong_versions[page.version-index].ce-version}}_amd64.deb){:.install-listing-link}
>  </span>
```

## Prerequisites

You have a supported system with root or [root-equivalent](/gateway/{{page.kong_version}}/plan-and-deploy/kong-user) access.

## Install

> You can install Gateway by downloading an installation package or using our `apt` repository.

Reviewer: ```suggestion
You can install Gateway by downloading an installation package or using our APT repository.
```

{% navtabs %}

> {% navtab Install with packages %}

Reviewer: ```suggestion
{% navtab Package %}
```

> {% navtabs %}
> {% navtab Install with packages %}
> 
>     {% if include.distribution == "ubuntu" %}
> 
> - [Xenial]({{ site.links.download }}/gateway-2.x-ubuntu-xenial/pool/all/k/kong/kong_{{site.data.kong_latest.version}}_amd64.deb)
> - [Bionic]({{ site.links.download }}/gateway-2.x-ubuntu-bionic/pool/all/k/kong/kong_{{site.data.kong_latest.version}}_amd64.deb)
> - [Focal]({{ site.links.download }}/gateway-2.x-ubuntu-focal/pool/all/k/kong/kong_{{site.data.kong_latest.version}}_amd64.deb)
> 
>     {% endif %}
> 
>     {% if include.distribution == "debian" %}
> 
> - [8 Jessie]({{ site.links.download }}/gateway-2.x-debian-jessie/pool/all/k/kong/kong_{{site.data.kong_latest.version}}_amd64.deb)
> - [9 Stretch]({{ site.links.download }}/gateway-2.x-debian-stretch/pool/all/k/kong/kong_{{site.data.kong_latest.version}}_amd64.deb)
> - [10 Buster]({{ site.links.download }}/gateway-2.x-debian-buster/pool/all/k/kong/kong_{{site.data.kong_latest.version}}_amd64.deb)
> - [11 Bullseye]({{ site.links.download }}/gateway-2.x-debian-bullseye/pool/all/k/kong/kong_{{site.data.kong_latest.version}}_amd64.deb)
> 
>     {% endif %}

Reviewer: ```suggestion
```
Moved all of this into the banner at the top, and removed Ubuntu links, since those got moved to the Ubuntu topic.

> Install Debian from the command line.
> 
> ```bash
> curl -Lo kong.{{site.data.kong_latest.version}}.amd64.deb "{{ site.links.download }}/gateway-2.x-{{ include.distribution }}-$(lsb_release -cs)/pool/all/k/kong/kong_{{site.data.kong_latest.version}}_amd64.deb"
> ```
> 
> ```bash
> sudo dpkg -i kong.{{site.data.kong_latest.version}}.amd64.deb
> ```

Reviewer: ```suggestion
Install {{site.base_gateway}} on Debian from the command line.

1. Download the {{site.base_gateway}} package:
    ```bash
    curl -Lo kong.{{page.kong_versions[page.version-index].ce-version}}.amd64.deb "{{ site.links.download }}/gateway-2.x-debian-$(lsb_release -cs)/pool/all/k/kong/kong_{{page.kong_versions[page.version-index].ce-version}}_amd64.deb"
     ```

2. Install the package:
    ```bash
    sudo dpkg -i kong.{{site.data.kong_latest.version}}.amd64.deb
    ```
```

{% endnavtab %}

> {% navtab Install with apt repository %}

Reviewer: ```suggestion
{% navtab APT repository %}
```

> Install the `apt` repository from the command line.

Reviewer: ```suggestion
Install the APT repository from the command line.
```

> ```bash
> echo "deb [trusted=yes] {{ site.links.download }}/gateway-2.x-{{ include.distribution }}-$(lsb_release -sc)/ default all" | sudo tee /etc/apt/sources.list.d/kong.list 
> sudo apt-get update
> sudo apt install -y kong
> ```

Reviewer: ```suggestion
1. Download the {{site.ce_product_name}} APT repository:
    ```bash
    echo "deb [trusted=yes] {{ site.links.download }}/gateway-2.x-debian-$(lsb_release -sc)/ 
    default all" | sudo tee /etc/apt/sources.list.d/kong.list 
    ```
2. Update the repository:
    ```bash
    sudo apt-get update
    ```
3. Install {{site.ce_product_name}}:
    ```bash
    apt install -y kong
    ```
```
